src/core/logging/logger.ts

```diff
@@ -0,0 +1,61 @@
+import { BotConfig } from "../types/base/botConfig";
+import { LogType } from "../types/base/logging";
+import { DiscordLogger } from "./discordLogger";
+import { SlackLogger } from "./slackLogger";
+
+/**
+ *
+ */
+export class Logger {
+	private botConfig: BotConfig;
+	public discordLogger?: DiscordLogger;
+	public slackLogger?: SlackLogger;
+
+	// Codes that are not sent to external sources (discord, slack)
+	private externalExemptCodes: Array<number> = [4, 5, 6, 7, 8];
```

Comment:
The logger will always dump to console.  That array indicates the error codes that would not be sent to external loggers (discord, slack, etc.).  The thought is to not swamp an external channel with common errors.  I personally only want to see if they bot is successful, restarting or a major issue. I can always dig into the console logs in the container for more detail.

I did think it might make sense make it an env variable.  Just didn't get a chance to do that yet.

Reply:
then i'd probably at least remove 7 from it, means we lose trade based on auction size, means bidrate is too low or something, this is something I wanna act on immediately. 8 is debatable but I agree with the rest.